src/cf-typography/usage.md

```diff
@@ -57,18 +121,20 @@
 
 ### Large pull quote
 
-<aside class="pull-quote pull-quote__large">
-    <div class="pull-quote_body">
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-        Cum corrupti tempora nam nihil qui mollitia consectetur
-        corporis nemo culpa dolorum!
-    </div>
-    <footer>
-        <cite class="pull-quote_citation">
-            - Author Name
-        </cite>
-    <footer>
-</aside>
+<div class="block">
+    <aside class="pull-quote pull-quote__large">
+        <div class="pull-quote_body">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Cum corrupti tempora nam nihil qui mollitia consectetur
+            corporis nemo culpa dolorum!
+        </div>
+        <footer>
+            <cite class="pull-quote_citation">
+                - Author Name
+            </cite>
+        </footer>
+    </aside>
+</div>
```

Comment:
Oh, nevermind, I was thinking that this was the example code, but this is the code that will output the live example in the docs. In that case, whatever you think looks good!